tests/js/preferencesRouter-tests.js

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = fluid || require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+var jqUnit = require("jqUnit");
+var kettle = require("kettle");
+
+kettle.loadTestingSupport();
+
+require("../../src/js/firstDiscoveryServer.js");
+// Once https://issues.gpii.net/browse/GPII-1302 has been addressed
+// the following blind include should be replaced by a more robust
+// inclusion mechanism.
+require("../../node_modules/gpii-express/tests/js/lib/test-helpers.js");
```

Comment:
My understanding of the node require system suggests that this include can already be replaced by 

require("gpii-express/tests/js/lib/test-helpers.js")